ViewController.m

```diff
@@ -0,0 +1,212 @@
+        roundNoLabel = UILabel(frame:CGRect.zero);
+        roundNoLabel?.translatesAutoresizingMaskIntoConstraints = false;
+        roundNoLabel?.textAlignment = .center;
+        roundNoLabel?.text="";
+        view.addSubview(roundNoLabel!);
+        roundNoLabel?.font = UIFont(name: "Halvetica", size: 30)
+        
+        
+        
+        //Adding 'info' label
+        //let infoLabel=UILabel(frame:CGRect(x:70,y:650,width:200,height:80));
+        infoLabel=UILabel(frame:CGRect.zero);
+        infoLabel?.translatesAutoresizingMaskIntoConstraints = false;
+        infoLabel?.textAlignment = .center;
+        infoLabel?.text="info";
+        view.addSubview(infoLabel!);
+        infoLabel?.font = UIFont(name: "Halvetica", size: 17)
+        
+        self.applyConstraints();
+    }
+    
+    func applyConstraints(){
+        var constraints:[NSLayoutConstraint] = [];
+        slider?.translatesAutoresizingMaskIntoConstraints = false;
+        constraints.append((slider?.topAnchor.constraint(equalTo:view.safeAreaLayoutGuide.topAnchor,constant: 100))!);
+        constraints.append((slider?.centerXAnchor.constraint(equalTo: view.safeAreaLayoutGuide.centerXAnchor))!);
+        constraints.append((slider?.widthAnchor.constraint(equalToConstant: 300))!);
+        constraints.append((slider?.heightAnchor.constraint(equalToConstant: 20))!);
+        
+        
+        tryToHitLabel?.translatesAutoresizingMaskIntoConstraints = false;
+        
+        constraints.append((tryToHitLabel?.topAnchor.constraint(equalTo:slider!.bottomAnchor, constant:50))!);
+        constraints.append((tryToHitLabel?.centerXAnchor.constraint(equalTo:view.safeAreaLayoutGuide.centerXAnchor))!);
+        constraints.append((tryToHitLabel?.widthAnchor.constraint(equalToConstant: 300))!);
+        constraints.append((tryToHitLabel?.heightAnchor.constraint(equalToConstant: 20))!);
+        
+        constraints.append((myButton?.topAnchor.constraint(equalTo: tryToHitLabel!.bottomAnchor, constant:50))!);
+        constraints.append((myButton?.centerXAnchor.constraint(equalTo:view.safeAreaLayoutGuide.centerXAnchor))!);
+        constraints.append((myButton?.widthAnchor.constraint(equalToConstant: 200))!);
+        constraints.append((myButton?.heightAnchor.constraint(equalToConstant: 80))!);
+        
+        constraints.append((scoreLabel?.topAnchor.constraint(equalTo: myButton!.bottomAnchor, constant: 30))!);
+        constraints.append((scoreLabel?.centerXAnchor.constraint(equalTo:view.safeAreaLayoutGuide.centerXAnchor))!);
+        constraints.append((scoreLabel?.widthAnchor.constraint(equalToConstant:200))!);
+        constraints.append((scoreLabel?.heightAnchor.constraint(equalToConstant: 80))!);
+        
+        
+        constraints.append((scoreValueLabel?.topAnchor.constraint(equalTo: myButton!.bottomAnchor, constant: 30))!);
+        constraints.append((scoreValueLabel?.leadingAnchor.constraint(equalTo: scoreLabel!.trailingAnchor, constant: -50))!);
```

Comment:
Since you have not defined leading/trailing anchors for `scoreLabel`, this constraint will break when scoreLabel's text length is large.